plugin/input/dmesg/dmesg.go

```diff
@@ -136,3 +136,8 @@ func (p *Plugin) Commit(event *pipeline.Event) {
 		p.logger.Error("can't save offset file: %s", err.Error())
 	}
 }
+
+// PassEvent decides pass or discard event.
+func (p *Plugin) IsPassedEvent(event *pipeline.Event) bool {
```

Comment:
PassEvent or IsPassedEvent ?

Reply:
Fixed that naming: linux based file leak on Mac